doc/release-notes-24806.md

```diff
@@ -0,0 +1,5 @@
+## RPC changes
+
+- `getblockfrompeer`, introduced in v21.0.0, named argument `block_hash` has been
+  renamed `blockhash` to be aligned with the rest of the codebase. This is a breaking
+  change if using named parameters. (bitcoin#24806)
```

Comment:
nit: 
```suggestion
  change if using named parameters. (#6149)
```

Reply:
#6149 can also point to bitcoin#6149 so let's be precise
```suggestion
  change if using named parameters. (dash#6149)
```

Reply:
I don't think it's important to specify dash here. It's in our release notes after all :)

Reply:
Well, bitcoin devs add their repo prefix. I guess there might be a good reason for this.